src/app/components/help/help.component.html

```diff
@@ -50,6 +50,11 @@
 		<ion-label>EthStaker Knowledgebase</ion-label>
 	</ion-item>
 
+	<ion-item lines="none" (click)="openBrowser('https://launchpad.ethereum.org/en/withdrawals')">
+		<ion-icon slot="start" name="bulb-outline"></ion-icon>
+		<ion-label>Guide: Enable withdrawals</ion-label>
```

Comment:
**Minor:** This is the only entry that starts with "Guide: ":
![image](https://user-images.githubusercontent.com/114066135/227911773-033b05e8-02c9-47bb-8e9d-5d401e31aeb3.png)

Thoughts?

Reply:
Thanks, changed it to Enable Withdrawals Guide